ffw/UIRPC.js

```diff
@@ -391,6 +391,12 @@ FFW.UI = FFW.RPCObserver.create(
                 SDL.ResetTimeoutPopUp.resetTimeoutRPCs.removeObject('VR.PerformInteraction');
               }
               array['UI.PerformInteraction'] =  request.params.timeout/1000 * increase_value;
+
+              if (request.params && request.params.choiceSet == null) {
+                console.log('remove UI');
```

Comment:
@LitvinenkoIra looks like a debug message, please remove it

Reply:
Done in d70c4290f1d80552bd0f5febdf197873f436002c